injected/integration-test/page-objects/broker-protection.js

```diff
@@ -52,19 +52,21 @@ export class BrokerProtectionPage {
      * @return {Promise<void>}
      */
     async isFormFilled () {
-        await expect(this.page.getByLabel('First Name:')).toHaveValue('John')
-        await expect(this.page.getByLabel('Last Name:')).toHaveValue('Smith')
-        await expect(this.page.getByLabel('Phone Number:')).toHaveValue(/^\d{10}$/)
-        await expect(this.page.getByLabel('Street Address:')).toHaveValue(/^\d+ [A-Za-z]+(?: [A-Za-z]+)?$/)
-        await expect(this.page.getByLabel('State:')).toHaveValue('IL')
-        await expect(this.page.getByLabel('Zip Code:')).toHaveValue(/^\d{5}$/)
+        await expect(this.page.getByLabel('First Name:', { exact: true })).toHaveValue('John')
+        await expect(this.page.getByLabel('Last Name:', { exact: true })).toHaveValue('Smith')
+        await expect(this.page.getByLabel('Phone Number:', { exact: true })).toHaveValue(/^\d{10}$/)
+        await expect(this.page.getByLabel('Street Address:', { exact: true })).toHaveValue(/^\d+ [A-Za-z]+(?: [A-Za-z]+)?$/)
+        await expect(this.page.locator('#state')).toHaveValue('IL')
```

Comment:
This would never be set for some reason when using just 'State:', so I changed it to just search by `id` instead.